test/general-use.js

```diff
@@ -33,17 +33,22 @@ module.exports = {
     english: {
 
         // english (appropriate mode) (sorted)
+        "Jack": "anga;quesse:a,tilde-below",
+        "happy": "hyarmen;parma:a,tilde-below;long-carrier:y",
+        "style": "silme;tinco;lambe:y,i-below",
+        "yellow": "anna;lambe:e,tilde-below;vala:o",
+        "phone": "formenparma;numen:o,i-below",
         "cake": "quesse;quesse:a,i-below",
         "cakes": "quesse;quesse:a;silme-nuquerna:e",
         "cats.": "quesse;tinco:a,s-final;full-stop", // regression
-        "green": "ungwe;romen;long-carrier:e;numen",
+        "green": "ungwe;romen;short-carrier:e;numen:e",
```

Comment:
Are there any cases where we need to preserve the long carrier for English in the mode for general use? This will require changes in `general-use.js`.

Reply:
We don't specifically need the long carrier at all for English. Tolkien sometimes used it for pronounced vowels at the ends of words, like the Y in "by" and "history" in the title page inscription. But, other times he didn't use a long carrier at all. So, for General Use, there is no pressing need for there to be a long carrier.

I like to use it at the ends of words because I think it's pretty though. But, that's just my personal preference.

Reply:
We might be able to preserve the long carrier for final Y æsthetic. Thank you for the details.